pyside6_overlay/core/game_detector.py

```
@@ -0,0 +1,156 @@
+            return current_rect
+            
+        except Exception as e:
+            print(f"Error getting window rect: {e}")
+            return None
+            
+    def is_window_minimized(self, hwnd: int) -> bool:
+        """Check if window is minimized"""
+        try:
+            placement = win32gui.GetWindowPlacement(hwnd)
+            return placement[1] == win32con.SW_SHOWMINIMIZED
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)